src/components/OverflowModals/DeleteTavleModal.tsx

```diff
@@ -7,26 +7,31 @@ import { useToast } from '@entur/alert'
 import sikkerhetBom from '../../assets/images/sikkerhet_bom.png'
 import retinaSikkerhetBom from '../../assets/images/sikkerhet_bom@2x.png'
 import { deleteTavle } from '../../settings/FirestoreStorage'
-import { CloseButton } from './LoginModal/CloseButton/CloseButton'
-import './Modals.scss'
+import { CloseButton } from '../CloseButton/CloseButton'
+import './OverflowModals.scss'
 
-const DeleteTavleModal = ({ open, onDismiss, id }: Props): JSX.Element => {
+interface DeleteTavleModalProps {
+    open: boolean
+    onDismiss: () => void
+    id: string
+}
+
+const DeleteTavleModal: React.FC<DeleteTavleModalProps> = ({
+    open,
+    onDismiss,
+    id,
+}) => {
     const { addToast } = useToast()
-    const overflowDeleteTavle = useCallback(
-        (remove: boolean) => {
-            if (remove) {
-                deleteTavle(id)
-                addToast({
-                    title: 'Avgangstavla ble slettet.',
-                    content:
-                        'Tavla ble slettet permanent og er dermed ikke lenger tilgjengelig.',
-                    variant: 'success',
-                })
-            }
-            onDismiss()
-        },
-        [id, onDismiss, addToast],
-    )
+    const handleDelete = useCallback(() => {
+        deleteTavle(id)
```

Comment:
Ser ut som det er litt inconsistent bruk av async fra funksjonene i FireStorage.ts, har det noe å si? vet ikke nok om fordeler og ulemper med async til å si noe om det

Reply:
Tror det kan ha noe å si om async funksjonen feiler. Hadde sikkert vært bedre å brukt .then og .catch for å håndtere de forskjellige situasjonen som kan oppstå.